Apps/Scigraph/dwim/draw.lisp

```diff
@@ -268,43 +172,17 @@ advised of the possiblity of such damages.
 		    (thickness 0) &allow-other-keys)
   ;; 30 Sep 91.  CLIM 1.0 bug was detected for start-angle and end-angle.
   ;; Hence won't pass those keywords along unless supplied.
-  #+clim (if (or start-p end-p)
+  (if (or start-p end-p)
 	     (clim:draw-circle* stream u v radius :ink alu :filled filled
 				:start-angle start-angle :end-angle end-angle
 				:line-thickness thickness )
 	     (clim:draw-circle* stream u v radius :ink alu :filled filled
-				:line-thickness thickness))
-  #-clim (graphics:draw-circle u v radius :stream stream :alu alu
-			       :filled filled :thickness thickness
-			       :start-angle start-angle :end-angle end-angle))
+				:line-thickness thickness)))
 
 (defun DRAW-RECTANGLE (left right bottom top &key (stream *standard-output*) (alu %alu)
 		       (filled nil) (thickness nil) &allow-other-keys)
-  #+clim (clim:draw-rectangle* stream left top (1+ right) (1+ bottom)
-			       :ink alu :filled filled :line-thickness thickness)
-  #-clim (graphics:draw-rectangle left top right bottom :stream stream :alu alu
-				  :filled filled :thickness thickness))
+  (clim:draw-rectangle* stream left top (1+ right) (1+ bottom)
+                        :ink alu :filled filled :line-thickness thickness))
 
-(defun draw-ellipse (x-center y-center x-radius y-radius &key
```

Comment:
Yeah, we don't need dwim:draw-ellipse, AFAICT. And the other dwim drawing functions will also go away over time.